twitterscraper/query.py

```diff
@@ -32,21 +32,20 @@ def query_single_page(url, html_response=True, retry=10):
 
     try:
         response = requests.get(url, headers=headers)
-        if html_response:
-            html = response.text
+        if resonse.status_code == 200:
+            if html_response:
+                html = response.text
+                tweets = list(Tweet.from_html(html))
+                return tweets, "TWEET-{}-{}".format(tweets[-1].id, tweets[0].id)
+            else:
+                json_resp = response.json()
+                html = json_resp['items_html']
+                tweets = list(Tweet.from_html(html))
+                return tweets, json_resp['min_position']
         else:
```

Comment:
we should have handling for non 200 error codes IMO, I usually find failing silently with no results letting the user believe everything went well is rather dangerous